### PR TITLE
[bitnami/parse] Parse standarization

### PR DESCRIPTION
**Pull request by fmulero**
Signed-off-by: Fran Mulero <fmulero@vmware.com>

**Description of the change**

This PR updates mongodb dependency and applies standardisation changes to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/parse/templates/ingress.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
             {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
             pathType: {{ .Values.ingress.dashboard.pathType }}
             {{- end }}
-            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "dashboard-http" "context" $)  | nindent 14 }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http-dahsboard" "context" $)  | nindent 14 }}
```
Comment (FraPazGal) [inline bitnami/parse/templates/ingress.yaml:32]:
```suggestion
            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http-dashboard" "context" $)  | nindent 14 }}
```

**Review by FraPazGal — CHANGES_REQUESTED** [review]
I added some suggestions and a couple of concerns, please take a look at them.

Additionally, regarding the use of a new MongoDB major, we need to update the `mongodb.persistence.accessMode` param as it is now `mongodb.persistence.accessModes` and uses a different format.

**Comment by FraPazGal** [comment]
Please take into account the need to update parse's [vib pipeline](https://github.com/bitnami/charts/blob/master/.vib/parse/vib-verify.json) as you changed the `service.port` definition. You may need to rebase from master to have said fail in your branch.

File: bitnami/parse/values.yaml [hunk 2]
```diff
@@ -166,6 +241,30 @@ server:
...
     failureThreshold: 5
+  ## @param server.startupProbe.enabled Enable startupProbe on Parse containers
+  ## @param server.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param server.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param server.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param server.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param server.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: true
```
[the remark this answers is not in the document]
Comment (fmulero) [reply bitnami/parse/values.yaml]: I left it as `true` during my tests

File: bitnami/parse/values.yaml [hunk 3]
```diff
@@ -389,20 +567,47 @@ dashboard:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+    ## Enable debug mode
+    ##
+    debug: false
+  ## @param dashboard.replicaCount Number of Parse Dashboard replicas to deploy
+  ##
+  replicaCount: 1
```
[the remark this answers is not in the document]
Comment (fmulero) [reply bitnami/parse/values.yaml:372]: No issues detected (at the moment). `sessionAffinity` needs to be configured to allow using multiple dashboard replicas properly

File: bitnami/parse/Chart.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@ appVersion: 5.2.0
 dependencies:
   - name: mongodb
     repository: https://charts.bitnami.com/bitnami
-    version: 11.x.x
+    version: ">=11, <=12"
```
Comment (migruiz4) [inline bitnami/parse/Chart.yaml:8]:
```suggestion
    version: "12.x.x"
```

File: bitnami/parse/README.md [hunk 5]
```diff
@@ -382,7 +458,21 @@ This version standardizes the way of defining Ingress rules. When configuring a
...
 
-### To 15.0.0
+### To 18.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be inline with the rest of assets in the Bitnami charts repository.
+
+Affected values:
+
+- `server.port` is renamed to `server.containerPorts.http`
+- `service.port` is renamed to `service.ports.mongodb`
```
Comment (migruiz4) [inline bitnami/parse/README.md:468]: This is a typo, it should be `service.ports.http`, right?

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Two minor changes requested, apart from that looks good to me

**Review by migruiz4 — DISMISSED** [review]
LGTM

File: bitnami/parse/README.md [hunk 6]
```diff
@@ -343,6 +430,26 @@ Find more information about how to deal with common errors related to Bitnami's
...
+### To 18.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be inline with the rest of assets in the Bitnami charts repository.
+
+Affected values:
+
+- `server.port` is renamed to `server.containerPorts.http`
+- `service.port` is renamed to `service.dashboard.ports.http`
+- `metrics.containerPort` is renamed to `metrics.containerPorts.metrics`
+- `service.nodePort` is renamed to `service.dashboard.nodePorts.http`
```
Comment (migruiz4) [inline bitnami/parse/README.md:442]: Here you mention `service.dashboard.*`, but the value is `dashboard.service` if I'm not mistaken.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Overall looks good to me, just a typo in the README.md

**Review by migruiz4 — APPROVED** [review]
LGTM!

**Review by migruiz4 — DISMISSED** [review]

**Review by migruiz4 — APPROVED** [review]

**Pull request merged**